guides/common/modules/con_using-pxe-to-provision-hosts.adoc

```diff
@@ -42,12 +42,31 @@ endif::[]
 
 In {Project} provisioning, the PXE loader option defines the DHCP `filename` option to use during provisioning.
 For BIOS systems, use the *PXELinux BIOS* option to enable a provisioned node to download the `pxelinux.0` file over TFTP.
-For UEFI systems, use the *PXEGrub2 UEFI* option to enable a TFTP client to download `grub2/grubx64.efi` file, or use the *PXEGrub2 UEFI HTTP* option to enable an UEFI HTTP client to download `grubx64.efi` from {SmartProxy} with the HTTP Boot feature.
+For UEFI systems, use the *Grub2 UEFI* option to enable a TFTP client to download `grub2/grubx64.efi` file, or use the *Grub2 UEFI HTTP* option to enable an UEFI HTTP client to download `grubx64.efi` from {SmartProxy} with the HTTP Boot feature.
 ifndef::satellite[]
 Use SecureBoot options to enable a client to download the `shim.efi` bootstrap bootloader that then loads the signed `grubx64.efi`.
+endif::[]
+
+ifndef::orcharhino,satellite[]
 Other PXE loaders like PXELinux UEFI, Grub2 ELF or iPXE Chain, require additional configuration. These workflows are not documented at the moment.
 endif::[]
 
 For BIOS provisioning, you must associate a PXELinux template with the operating system.
 For UEFI provisioning, you must associate a PXEGrub2 template with the operating system.
 If you associate both PXELinux and PXEGrub2 templates, {Project} can deploy configuration files for both on a TFTP server, so that you can switch between PXE loaders easily.
+ifndef::satellite[]
+
+.Provisioning of Secure Boot enabled hosts
+By default, you can provision Secure Boot enabled hosts that match the operating system of your {ProjectServer}.
+You have to use the SecureBoot variants of the *Grub2 UEFI* and *Grub2 UEFI HTTP* PXE loaders to enable a client to download the `shim.efi` bootstrap bootloader that then loads the signed `grubx64.efi`.
```

Comment:
Global recommendation: Change "You have" to "You must" to align with the IBM Style Guide. 

"You have" can create problems with localization and translations into other languages because "have" has many different meanings.

Reply:
Fixed